### DISCUSSION TITLE
Hello author, please ask which version can use YOLOV5 6.1 or 6.2 to do detection to achieve the purpose of tracking

### DISCUSSION THREAD

**Discussion by JMChenl** [ultralytics/yolov5#9019] [Q&A]

**Comment by glenn-jocher** [comment]
@JMChenl 👋 Hello! Thanks for asking about object **tracking** in computer vision. YOLOv5 🚀 is an object **detector** that detect, localizes and classifies objects in a single image. It does not connect objects across multiple images, for this you need a tracking solution. A few possible tracking solutions are:

- [Extended Kalman Filter (EKF)](https://en.wikipedia.org/wiki/Kalman_filter)
- [Kanade-Lukas-Tomasi (KLT) Feature Tracker](https://en.wikipedia.org/wiki/Kanade%E2%80%93Lucas%E2%80%93Tomasi_feature_tracker)
- [YOLOv5 DeepSort Tracker](https://github.com/mikel-brostrom/Yolov5_DeepSort_Pytorch) 
- [Apple iOS VNTrackObjectRequest](https://developer.apple.com/documentation/vision/tracking_multiple_objects_or_rectangles_in_video)

Good luck 🍀 and let us know if you have any other questions!

**Comment by JMChenl** [comment]
Thank you very much for answering my question, YOLOV5 is one of the best object detection algorithms I have ever used, thank you very much for your help to me.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Glenn ***@***.***&gt;; 
发送时间: 2022年8月19日(星期五) 凌晨2:58
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ultralytics/yolov5] Hello author, please ask which version can use YOLOV5 6.1 or 6.2 to do detection to achieve the purpose of tracking (Discussion #9019)






@JMChenl👋你们好！谢谢询问对象跟踪在计算机视觉中。YOLOv5🚀是一个对象探测器用于检测、定位和分类单个图像中的对象。它不连接多个图像的对象，为此您需要一个跟踪解决方案。一些可能的跟踪解决方案是:

扩展卡尔曼滤波（EKF）

Kanade-Lukas-Tomasi (KLT) 特征跟踪器

YOLOv5 DeepSort追踪器

Apple iOS VNTrackObjectRequest

祝你好运🍀如果您还有其他问题，请告诉我们！

—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.
